controllers/datadogagent/component/agent/default.go

```diff
@@ -176,6 +187,12 @@ func systemProbeContainer(dda metav1.Object) corev1.Container {
 		VolumeMounts:   volumeMountsForSystemProbe(),
 		LivenessProbe:  apicommon.GetDefaultLivenessProbe(),
 		ReadinessProbe: apicommon.GetDefaultReadinessProbe(),
+		SecurityContext: &corev1.SecurityContext{
+			SeccompProfile: &corev1.SeccompProfile{
```

Comment:
Based on the agent helm chart implementation, there looked to always be a seccomp profile configured for the system probe container. This can be overridden with a different profile in the `SecurityContext` section of the overrides